change/@ni-nimble-components-d62c8a19-26e8-448d-831b-ddc56969c9bb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If I do a text search for colorsscale I still find references in README.md, data-manager.spec.ts, types.spec.ts, utilities.ts, and wafer-map.stories.ts

Could you make sure those locations are cleaned up as well?